WordPress/src/main/java/org/wordpress/android/ui/DualPaneFragment.java

```diff
@@ -0,0 +1,31 @@
+package org.wordpress.android.ui;
+
+import android.os.Build;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.View;
+
+import org.wordpress.android.R;
+
+/**
+ * Superclass used for shared logic of fragments with dual pane layout support.
+ */
+public class DualPaneFragment extends Fragment {
```

Comment:
I reworked concept a bit after this PR, so this part have changed. In short, no, you don't have to to descend from DualPaneFragment. In fact all of it's code have moved to as simple helper class. Dual pane "host" fragment should not care about how it's direct children (sidebar and content) are implemented, because he does not directly interacts with them (except for adding/removing). It exposes some of it's functionality through interface, and let's sidebar and content fragments decide what do they want to do with it.